numba_dppy/tests/integration/test_dpnp_interop.py

```diff
@@ -17,7 +17,8 @@
 import pytest
 
 import numba_dppy as dppy
-from numba_dppy.tests._helper import ensure_dpnp, skip_test
+from numba_dppy.tests._helper import filter_strings
+from numba_dppy.tests.njit_tests.dpnp.dpnp_skip_test import skip_no_dpnp
```

Comment:
Instead of creating `dpnp_skip_test.py` and importing `skip_no_dpnp` for each function you could just skip all tests on module level `.../dpnp/__init__py`.
See as example: https://github.com/IntelPython/numba-dppy/blob/1ec2809a5cada72bdf136da4d72763375a983929/numba_dppy/tests/test_debug_dppy_numba.py#L28